extension/gulpfile.babel.js

```diff
@@ -89,17 +89,17 @@ gulp.task('babel', () => {
     'app/scripts.babel/app.js',
     'app/scripts.babel/chromereload.js',
     'app/scripts.babel/background.js'])
-    .pipe($.sourcemaps.init())
+    // .pipe($.sourcemaps.init())
```

Comment:
just delete? Our transformation isn't complex, and in theory we don't want any more than the module concat anyways, so I don't think we'll want it back